test-cypress/integration/explore/class.hierarchy.spec.js

```diff
@@ -172,7 +172,7 @@ describe('Class hierarchy screen validation', () => {
         getReturnButton().should('be.visible').click();
     });
 
-    it('Test class-hierarchy for given graph', () => {
+    it.skip('Test class-hierarchy for given graph', () => {
```

Comment:
Will be better if we move this test in flake suite, so we can run it locally.

Reply:
Just a test I will revert it